godtools/App/Share/Data/RealmDatabase/RealmDatabaseConfiguration+RealmConfig.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  RealmDatabaseConfiguration+RealmConfig.swift
+//  godtools
+//
+//  Created by Levi Eggert on 11/14/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+extension RealmDatabaseConfiguration {
+    
+    func getRealmConfig() -> Realm.Configuration {
+        
+        var realmConfig: Realm.Configuration
+        
+        switch cacheType {
+            
+        case .disk(let fileName, let migrationBlock):
+            
+            realmConfig = Realm.Configuration()
+            realmConfig.fileURL = realmConfig.fileURL?.deletingLastPathComponent().appendingPathComponent(fileName)
+            realmConfig.schemaVersion = schemaVersion
+            realmConfig.migrationBlock = migrationBlock
```

Comment:
Hey @rachaelblue I had to make one more change in this PR which was removing the MigrationBlock closure from the extension here and instead have that be injected with the RealmDatabaseCacheType disk.

The migrationBlock is now created in RealmDatabaseProductionConfiguration.
```
let migrationBlock = { (migration: Migration, oldSchemaVersion: UInt64) in

            if (oldSchemaVersion < 1) {
                // Nothing to do!
                // Realm will automatically detect new properties and removed properties
                // And will update the schema on disk automatically
            }
        }
```